package.json

```
@@ -0,0 +1,29 @@
+{
+  "type": "module",
+  "name": "call_of_duty",
+  "version": "1.0.0",
+  "description": "Call Of Duty",
+  "main": "server.js",
+  "scripts": {
+    "test": "vitest --run --watch --silent --threads false",
+    "start": "node server.js",
```

You can add `pino-pretty` to start and dev to make your log formatted.
`npm i -D pino-pretty`
and then add:
```suggestion
    "start": "node server.js | pino-pretty",
```

---

done